connect.go

```diff
@@ -68,6 +70,37 @@ type ConfigurationsUpdate struct {
 	Update      bool   `json:"update"`
 }
 
+// FetchOpts - configuration options for fetch.
+type FetchOpts struct {
+	Name                     string
+	StationName              string
+	ConsumerGroup            string
+	BatchSize                int
+	BatchMaxTimeToWait       time.Duration
+	MaxAckTime               time.Duration
+	MaxMsgDeliveries         int
+	GenUniqueSuffix          bool
+	ErrHandler               ConsumerErrHandler
+	StartConsumeFromSequence uint64
+	LastMessages             int64
+}
+
+// getDefaultConsumerOptions - returns default configuration options for consumers.
+func getDefaultFetchOptions() FetchOpts {
+	return FetchOpts{
+		BatchMaxTimeToWait:       5 * time.Second,
```

Comment:
consumer group is also not mandatory default is "" and batch size is also can default to 10.
the batch size should only be mandatory on the fetch (of the consumer)